### PR TITLE
Add initial sections for docs site

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kim.andrewsy@gmail.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
We're trying to improve documentation on the AWS cloud provider. Before divvying up the work, I would like agreement on what the content should look like at a high level. I've added some initial sections that I think is worth covering, I'd like to use this PR to discuss what other content we should include.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by randomvariable** [comment]
lgtm as a skeleton to fill in.

File: docs/book/SUMMARY.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # Summary
 
 * [Introduction](README.md)
+* [Prerequisites](prerequisites.md)
+* [Getting Started](getting_started.md)
+* [Service LoadBalancers](service_loadbalancers.md)
+* [Known Limitations](known_limitations.md)
```
Comment (andrewsykim) [inline docs/book/SUMMARY.md:7]: What else do we need to add here?

**Comment by andrewsykim** [comment]
/hold

Pending discussion on doc template

File: docs/book/SUMMARY.md (hunk 1 above)
Comment (randomvariable) [reply docs/book/SUMMARY.md:7]: EBS is going off to CSI now?
IAM perms is usually the key thing.

Comment (randomvariable) [reply docs/book/SUMMARY.md:7]: Actually, tags might be helpful to document.

Comment (andrewsykim) [reply docs/book/SUMMARY.md:7]: > IAM perms is usually the key thing.

Agreed, I think this would go into the Prerequisites section.

Comment (andrewsykim) [reply docs/book/SUMMARY.md:7]: > Actually, any default tags the CPI set might be helpful to document.

Wondering what section this would belong to.

Comment (randomvariable) [reply docs/book/SUMMARY.md:7]: sgtm

**Comment by nckturner** [comment]
Let's merge this so we can start adding to it.

**Pull request merged**